### DISCUSSION TITLE
Save posts feature

### DISCUSSION THREAD

**Discussion by chunyu-li** [civitai/civitai#568] [Ideas]
So far I can like models and bookmark articles in civitai so that I can review them later, but I haven't found a button to save posts yet. Is there such feature in civitai? For example, I want to save or like an image post like this below:

<img width="1278" alt="Screenshot 2023-06-11 at 17 12 50" src="https://github.com/civitai/civitai/assets/61857563/d610b7a3-f7d9-42be-aae2-2e970fd953ba">

**Comment by PeterNjeim** [comment]
Indeed it seems you can't save posts currently. As a workaround for now, you can react to every image in a post and see them on your profile page's Images tab in the `My Reactions` subsection:
![image](https://github.com/civitai/civitai/assets/26418974/f4de4e30-60ec-4714-9152-b4f50c7965bf)

The `My Reactions` subsection is only visible to yourself, not to the public
